test/JDBC/expected/alter-procedure-schema.out

```diff
@@ -425,3 +425,33 @@ void
 -- tsql 
 drop login alter_proc_l4;
 go
+
+-- psql currentSchema=master_dbo,public
+-- Test psql procedures altered with security definer do not throw StartTransactionCommand: unexpected state STARTED error 
+-- Test alter procedure using tsql dialect in PSQL port throws error
+CREATE PROCEDURE master_dbo.p1() AS $$ BEGIN SELECT 1; END $$ LANGUAGE plpgsql;
+go
+
+alter procedure master_dbo.p1 security definer;
+go
+
+drop procedure master_dbo.p1;
+go
+
+set babelfishpg_tsql.sql_dialect = "tsql";
```

Comment:
We need to reset this. Since our JDBC test framework re-uses connections, it can affect other test cases.